flake.nix

```diff
@@ -2,112 +2,123 @@
   description = "The nixos.org homepage";
 
   # This is used to build the site.
-  inputs.nixpkgs.url = "nixpkgs/nixos-20.03";
+  inputs.nixpkgs = { url = "nixpkgs/nixos-20.03"; };
 
-  # These are used for the manuals, and release artifacts
-  inputs.released-nixpkgs.url = "nixpkgs/nixos-20.03";
+  # These inputs are used for the manuals and release artifacts
+  inputs.released-nixpkgs = { url = "nixpkgs/nixos-20.03"; };
+  inputs.released-nix = { url = "github:nixos/nix/2.3-maintenance"; flake = false; };
```

Comment:
A slight problem is that this branch also gets pushed to between releases (so it might point to an unreleased version). So it's not like the `nixos-*` branches which always correspond to a released version.

Reply:
So concretly, `NIX_VERSION` might end up pointing to a version that doesn't exist in releases.nixos.org yet.

Reply:
Would it be possible to change the Nix release workflow and add some sort of `2.3-release` branch?

Reply:
Maybe a `latest` branch that gets force-pushed when we do a new release. I can add this to the release script.